easyInterface/Diffraction/Interface.py

```diff
@@ -492,7 +526,7 @@ def setPhases(self, phases: Union[Phase, Phases]) -> NoReturn:
         """
         if isinstance(phases, Phase):
             new_phase_name = phases['phasename']
-            k, v = self.project_dict.getItemByPath(['phases', new_phase_name]).dictComparison(phases)
+            k, v, t = self.project_dict.getItemByPath(['phases', new_phase_name]).dictComparison(phases)
```

Comment:
Should we use a less cryptic names, such as `keys`, `values`? And what is `t`? It seems that `t` is not used.

Reply:
Modified